src/components/Questions.js

```diff
@@ -0,0 +1,82 @@
+import { useState, useEffect } from 'react';
+import Question from './Question';
+
+const Questions = () => {
+  const [data, setData] = useState([]);
+  const [loading, setLoading] = useState(false);
+  const [checked, setChecked] = useState(false);
+  const [success, setSuccess] = useState(Array.from({ length: 5 }));
+
+  const getData = async () => {
+    setLoading(true);
+    const res = await fetch('https://opentdb.com/api.php?amount=5');
+    const { results } = await res.json();
+    const questions = results.map((question, index) => ({
+      question: question.question,
+      correct: question.correct_answer,
+      answers: shuffledHandler(question.correct_answer, question.incorrect_answers),
+      id: index,
+    }));
+    setData(questions);
+    setLoading(false);
+  };
+
+  useEffect(() => {
+    getData();
+  }, []);
+
+  const shuffledHandler = (correct, incorrect) => {
+    const answers = [...[correct], ...incorrect];
+    const shuffled = answers.sort(() => Math.random() - 0.5);
+    const newArray = shuffled.map(item => item);
+    return newArray;
+  };
```

Comment:
- I am not getting why you need to extract the correct answers and then put what you just extracted in an array. I think you only need to extract the incorrect values only and leave the correct values as an array without extracting them. This will prevent you to have what's not needed. 
- Secondly, I believe you don't really need line 31. Mapping through an array and returning the same array, doesn't really do much. You can directly return the sorted array without first using the unnecessary map function.